assets/css/main.css

```diff
@@ -447,3 +447,30 @@ table.results-table .category-row {
     margin-top: 4rem;
     width: 80%;
 }
+
+/* Skip link stuff */
+
+.skip-link { /* This one is equally applicable to all skip links */
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    margin: 0;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    display: block;
+}
+
+.skip-link:focus {
+    position: static;
+    width: 100%;
+    min-height: 50px;
+    height: auto;
+    margin: inherit;
+    overflow: visible;
+    clip: auto;
+    padding: 15px;
+}
+
+.page-skip-link:focus {
+    min-height: 24px;
+}
```

Comment:
It would be nice if there were a new line in the end of the file. I look into adding editorconfig, so this would be automatically configured for everyone's editor.

Reply:
Marking as resolved, as it's covered by https://github.com/apluslms/a-plus/pull/565#discussion_r438571411